app/code/Magento/Ui/Setup/InstallSchema.php

```diff
@@ -64,10 +64,34 @@ public function install(SchemaSetupInterface $setup, ModuleContextInterface $con
                 ['nullable' => false],
                 'Mark current bookmark per user and identifier'
             )
-            ->addColumn('title', Table::TYPE_TEXT, 255, ['nullable' => true], 'Bookmark title')
-            ->addColumn('config', Table::TYPE_TEXT, Table::MAX_TEXT_SIZE, ['nullable' => true], 'Bookmark config')
-            ->addColumn('created_at', Table::TYPE_DATETIME, null, ['nullable' => false], 'Bookmark created at')
-            ->addColumn('updated_at', Table::TYPE_DATETIME, null, ['nullable' => false], 'Bookmark updated at')
+            ->addColumn(
```

Comment:
JFYI: this solution is not reliable since this will affect only new installations. This should have been done in `UpgradeSchema.php` with bumping module version.